python/lsst/dax/imgserv/cli.py

```diff
@@ -0,0 +1,162 @@
+#
+# LSST Data Management System
+# Copyright 2017 LSST/AURA.
+#
+# This product includes software developed by the
+# LSST Project (http://www.lsst.org/).
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the LSST License Statement and
+# the GNU General Public License along with this program.  If not,
+# see <http://www.lsstcorp.org/LegalNotices/>.
+
+'''
+This module implements the command line interface (CLI) to ImageServ.
+
+@author: Brian Van Klaveren, SLAC
+@author: Kenny Lo, SLAC
+
+'''
+
+import os
+import json
+
+import click
+
+import lsst.log as log
+import lsst.daf.persistence as dafPersist
+
+from .locateImage import (
+        image_open, W13DeepCoaddDb, W13RawDb, W13CalexpDb
+)
+
+from .image import Image
+from .dispatch import Dispatcher
+from .hashutil import Hasher
+
+ROOT = os.path.abspath(os.path.dirname(__file__))
+
+@click.command()
+@click.option("--config", help="Specify config file.")
+@click.option("--in_req", help="Specify query input.")
+@click.option("--out_dir", help="Specify the output directory.") 
+def exec_command(config_dir, in_req, out_dir):
+    """ Command Line Interface: Process query to return image file
+        in output directory.
+    """ 
+    cli = ImageServCLI(config_dir, in_req, out_dir)
+    cli.process_request()
+    
+class ImageServCLI(object):
+    """ Module to implement CLI for ImageServ.
+    
+    """
+    def __init__(self, config_dir, in_req, out_dir):
+        # load the configuration file
+        config = os.path.join(config_dir, "settings.json")
+        with open(config) as f:
+            self._config = json.load(f)
+            # strip /tests from dataroot if cur_dir is /tests
+            cwd = os.getcwd()
+            if "/tests" in cwd:
+                self._fix_dataroot_path()
+            f.close()
+         # configure the log file (log4cxx)
```

Comment:
Indentation. It might be good to make "/tests" a bit more specific to dax_img to avoid collisions. An explanation of the expected directory structure could be useful.